package.json

```diff
@@ -21,6 +21,7 @@
     "@types/react-modal": "3.16.0",
     "axios": "0.24.0",
     "bootstrap": "4.6.2",
+    "chart.js": "^4.4.0",
```

Comment:
Same here. Use 4.4.0 instead of ^4.4.0.

Using ^4.4.0 is like telling node: "use any package with a version higher than this" which is not what we want. We want to always know exactly what package version we are using in order to facilitate easy debugging